frontend/src/views/c-authorship.vue

```diff
@@ -202,6 +213,11 @@ function authorshipInitialState() {
     isIgnoredFilesChecked: false,
     searchBarValue: '',
     authorDisplayName: '',
+    authors: new Set(),
+    authorColors: {},
```

Comment:
```suggestion
```
Since the `authorColors` object is also stored in the Vuex store, would it make sense to use that state as the single source of truth for this object? (instead of having to maintain & sync both copies)

Reply:
While I initially went with that approach of assigning author colors whenever a new author is encountered in `splitSegments()`, I thought it would be better to first find all authors, and then assign colors separately for better separation of concerns. This would also remove the need for a global variable to track which author color is to be assigned next.

Hence, would you suggest we create an empty `authorColors` map while doing the segment split?

Reply:
I have made `authorColors` a local variable to have a single source of truth as per your suggestion!